src/dataProvider.js

```diff
@@ -683,9 +692,29 @@ function getServerHostname(chainName) {
   return 'gmx-server-mainnet.uw.r.appspot.com'
 }
 
+export function useVolumeDataRequest(url, defaultValue, from, to, fetcher = defaultFetcher) {
+  const [loading, setLoading] = useState(true)
```

Comment:
`useRequest` doesn't suit for some reason?

Reply:
I don't think this is even being used on any of the pages. We can remove this

Reply:
I think this is done to refetch on change of `to` or `from`.